cmd/nodef/genesiscmd.go

```diff
@@ -77,7 +78,14 @@ the base64 encoded publickey and a list of initial coins.`,
 				cdc.MustUnmarshalJSON(appState[eltypes.ModuleName], &genesisState)
 			}
 
-			genesisState.GenesisConf.Genesis.Accounts = append(genesisState.GenesisConf.Genesis.Accounts, account)
+			// check already existing address
+			for _, account := range genesisState.Accounts {
```

Comment:
The "account" in "for" iteration and incoming "account" should be different var name.
`Compare()` always says equal and `add-el-genesis-account` always fails.